appinventor/components/src/com/google/appinventor/components/runtime/WebViewer.java

```diff
@@ -384,7 +384,7 @@ public void UsesLocation(boolean uses) {
    * @return true if prompting is  required.  False assumes permission is granted.
    */
 
-  @SimpleProperty(description = "If True, then prompt the user of the WebView to give permission to access the geolocation API. " +
+  @SimpleProperty(description = "If true, then prompt the user of the WebView to give permission to access the geolocation API. " +
```

Comment:
This should (in my opinion) stay with capital, but if you are going to change it, it should be consistent and change also False in line 388).